transport/spi/src/main/java/org/kaazing/gateway/transport/LoggingFilter.java

```diff
@@ -94,6 +97,10 @@ public LoggingFilter(Logger logger) {
         this(logger, "%s");
     }
 
+    public LoggingFilter(Logger logger, IoSession session, String transportName) {
```

Comment:
Yes you are right. This is in line with all current usage, as you pointed out, so I think it is okay. An alternative would be to stash the format on a session attribute. But I think it's more straightforward to have an instance per session.